config/locales/en.yml

```diff
@@ -884,9 +884,10 @@ en:
     notice_template_added: Mail template was successfully added.
     notice_template_destroyed: Mail template was successfully destroyed.
     notice_template_updated: Mail template was successfully updated.
-    placeholders_hint: |
-      The following place holders may be used in the `content` section of each
-      template.
+    placeholders_hint_header: |
+      You can use dynamic variable substation in the in the `subject` and `content` sections of each
```

Comment:
I don't get this translation. Was that auto generated from German?

Reply:
Ahh, I get it. That should be a "substitution", not a substation.

Reply:
Fix (in all languages)

https://github.com/frab/frab/blob/24b078ab6c8e677983166143f42bb921330ed6a5/config/locales/en.yml#L887-L890

https://github.com/frab/frab/blob/24b078ab6c8e677983166143f42bb921330ed6a5/config/locales/de.yml#L899-L902